File: lib/src/main/java/com/sha/formvalidator/validator/pattern/WebUrlValidator.java
package com.sha.formvalidator.validator.pattern;

import android.util.Patterns;

/**
 * Validates a web url in the format:
 * scheme + authority + path
 *
 */
public class WebUrlValidator extends PatternValidator {

    public WebUrlValidator(String errorMessage) {
        super(errorMessage, Patterns.WEB_URL);
    }

}

File: lib/core/src/main/java/com/sha/formvalidator/textview/validator/pattern/WebUrlValidator.kt
package com.sha.formvalidator.textview.validator.pattern

import android.util.Patterns

/**
 * Validates a web url in the format:
 * scheme + authority + path
 *
 */
class WebUrlValidator(errorMessage: String) : PatternValidator(errorMessage, Patterns.WEB_URL)
